cedar-policy-core/src/parser/cst_to_ast.rs

```diff
@@ -211,7 +211,7 @@ impl ASTNode<Option<cst::Policy>> {
 
         for e in conds.iter() {
             for slot in e.slots() {
-                errs.push(ParseError::ToAST(format!("Template slots are currently unsupported in policy condition clauses, found slot {slot}")));
+                errs.push(ParseError::UnexpectedSlot(format!("Template slots are currently unsupported in policy condition clauses, found slot {slot}")));
```

Comment:
Can the explanation be in the `#[error]` annotation for this variant, and the argument just be the `{slot}`?